src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -194,6 +224,45 @@ DynamicInstrumentationMethod CaptureOptionsDialog::GetDynamicInstrumentationMeth
   ORBIT_UNREACHABLE();
 }
 
+void CaptureOptionsDialog::SetPureOrNotTracepoint(bool is_pure) {
```

Comment:
I have to admit `Pure` is a little confusing to me. May I suggest something more explicit?

What about `SetEnableCallStackCollectionOnThreadStateChanges`? (Ok, it's long. Maybe you have a better suggestion. :innocent: )

Reply:
Done.